optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/testgen/mutation/TestGenRemoveRandomBlockMutator.java

```diff
@@ -42,13 +42,13 @@ public boolean canMutate() {
         }
 
         if (removedIndex >= 0) {
-            // last mutation was successful => clear the blacklist
+            // last mutation was successful => clear the denylist
             indexBlacklist.clear();
```

Comment:
The variable should be renamed as well. See 75057b149703b4c7749eac639c3630cafd4e7e08.

Reply:
Good catch! Thanks. I checked for other combinations of capitals as well bb9900f9